docs/pages/machine-id/guides/hostcertificate.mdx

```diff
@@ -0,0 +1,403 @@
+Connect to the Teleport Auth Server and use `tctl` to examine what roles exist on
+your system.
+</ScopedBlock>
+
+```code
+$ tctl get roles --format=text
+```
+
+You will see something like the output below on a fresh install of Teleport with the
+default roles—your cluster may have different roles. In this example, let's
+assume you want to give the bot the `hostcert-bot` role to allow it to generate host certificates.
+
+```
+Role    Allowed to login as                           Node Labels Access to resources
+------- --------------------------------------------- ----------- ----------------------------------------
+access  {{internal.logins}}                           <all nodes> event:list,read,session:read,list
+auditor no-login-6566121f-b602-47f1-a118-c9c618ee5aec             session:list,read,event:list,read
+editor
+hostcert-bot  host_cert:create,list,read,update,delete             user:list,create,read,update,delete,...
+```
+
+Machine ID can join with a `token` or the [IAM Method](https://goteleport.com/docs/setup/guides/joining-nodes-aws) on AWS.
+
+While any role with permissions for logging into nodes will work, this guide will additionally make use of the default
+`access` role to ensure that the tbot user can successfully log in. Use the `--logins` flag when adding your bot to specify the SSH logins that you wish to 
+allow the bot to access on hosts. For our example, we will be using `root`.
+
+<Tabs>
+  <TabItem label="Token-based Joining">
+  ```code
+  $ tctl bots add robot --roles=hostcert-bot,access --logins=root
+  ```
+  </TabItem>
+  <TabItem label="IAM Method">
+  First, create an IAM method token that specifies the AWS account from which
+  the bot can join. Create the below file as `iam-token.yaml` then run `tctl
+  create -f iam-token.yaml`.
+
+  ```
+  kind: token
+  version: v2
+  metadata:
+    # The token name is not a secret because instances must prove that they are
+    # running in your AWS account to use this token.
+    name: iam-token
+    # Set a long expiry time for how long you want to support IAM method for
+    # joining. It is safe to set this value to a very long time.
+    expires: "3000-01-01T00:00:00Z"
+  spec:
+    # Only allow bots to join using this token.
+    roles: [Bot]
+
+    # Set the join method to be IAM.
+    join_method: iam
+
+    # Define the name of the bot that will be allowed to use this token.
+    bot_name: robot
+
+    allow:
+    # Restrict the AWS account and (optionally) ARN that can use this token.
+    # This information can be obtained from running the
+    # "aws sts get-caller-identity" command from the CLI.
+    - aws_account: "111111111111"
+      aws_arn: "arn:aws:sts::111111111111:assumed-role/teleport-bot-role/i-*"
+  ```
+
+  Next, create the bot user.
+
+  ```
+  $ tctl bots add robot --token=iam-token --roles=access --logins=root
+  ```
+  </TabItem>
+</Tabs>
+
+
+Machine ID may now be configured using the `tbot configure` command.
```

Comment:
Would it be worth mentioning these steps may be run on a different node? On our other guides we tend to differentiate using "node with `tctl` access" and "bot node", the latter being where `tbot` actually runs (and in this case, where `sshd` is also running).